packages/flutter/lib/src/cupertino/segmented_control.dart

```diff
@@ -158,24 +184,26 @@ class _SegmentedControlState<T> extends State<SegmentedControl<T>>
   final List<AnimationController> _selectionControllers = <AnimationController>[];
   final List<ColorTween> _childTweens = <ColorTween>[];
 
-  static final ColorTween forwardBackgroundColorTween = new ColorTween(
-    begin: _kPressedBackground,
-    end: CupertinoColors.activeBlue,
-  );
-
-  static final ColorTween reverseBackgroundColorTween = new ColorTween(
-    begin: CupertinoColors.white,
-    end: CupertinoColors.activeBlue,
-  );
-
-  static final ColorTween textColorTween = new ColorTween(
-    begin: CupertinoColors.activeBlue,
-    end: CupertinoColors.white,
-  );
+  static ColorTween forwardBackgroundColorTween;
```

Comment:
Should these still be static even though they are (re-) initialized in initState?

Also, do they have to be public? If so, they probably need a doc comment?